oauth2_http/java/com/google/auth/oauth2/ComputeEngineCredentials.java

```diff
@@ -82,6 +82,10 @@
   // "unlikely" since the expected 4-nines time is about 0.5 seconds.
   // This allows us to limit the total ping maximum timeout to 1.5 seconds
   // for developer desktop scenarios.
+  // However, this workaround doesn't work for GKE because GKE metadata resolution takes
+  // much longer time than 1.5 seconds. We won't have a timeout that works for all scenarios,
+  // so GKE users should use the `GCE_METADATA_TIMEOUT` environment variable to explicitly set the
+  // timeout in seconds.
   static final int MAX_COMPUTE_PING_TRIES = 3;
   static final int COMPUTE_PING_CONNECTION_TIMEOUT_MS = 500;
```

Comment:
Can we prepend `DEFAULT_` to this constant name to reflect that it's now a default.

Reply:
done